background.js

```diff
@@ -4,7 +4,7 @@
 browser.browserAction.onClicked.addListener(runLightbeam);
 
 async function runLightbeam() {
-
+  captureData();
```

Comment:
Because this is in the `runLightbeam` function, which is only called by `browserAction.onClicked`, no data is captured unless/until the user clicks the `browserAction` icon. Is that what we want?